x/proof/keeper/params_test.go

```diff
@@ -24,8 +24,8 @@ func TestParams_ValidateMinRelayDifficulty(t *testing.T) {
 	}{
 		{
 			desc:                   "invalid type",
```

Comment:
invalid type or invalid value?

Reply:
Invalid type. It should be a `uint64` but this test case uses `int64`.